Dockerfile

```diff
@@ -1,5 +1,8 @@
 FROM debian:stretch-slim
 
+COPY build_module.sh /
```

Comment:
Prefer `ADD`

Reply:
It would probably be better to move this instruction after the `RUN` below and split out the `chmod` in there. Most likely this would be a broken state after this instruction

Reply:
The current flow amounts to installing the script before its dependencies, so it is more sensible top move this new instruction below the deeps. After that the chmod should be split out as a separate RUN after that COPY

Reply:
Thanks Matt. I did understand earlier why `COPY` needs to be moved after `RUN` which installs dependencies. What I do not understand when you referred `Most likely this would be a broken state after this instruction`.